kyuubi-hive-jdbc/src/main/java/org/apache/kyuubi/jdbc/hive/JdbcConnectionParams.java

```diff
@@ -79,6 +79,7 @@ public class JdbcConnectionParams {
   // Use ZooKeeper for indirection while using dynamic service discovery
   static final String SERVICE_DISCOVERY_MODE_ZOOKEEPER = "zooKeeper";
   static final String ZOOKEEPER_NAMESPACE = "zooKeeperNamespace";
+  static final String ZOOKEEPER_STRATEGY = "zooKeeperStrategy";
```

Comment:
```suggestion
  static final String ZOOKEEPER_SELECT_POLICY = "zooKeeperSelectPolicy";
```

Is it a better param name here? cc @pan3793

Reply:
"select policy" is a better phrase here.

technically, the mechanism could be promoted to another service discovery impls, so we should not include "zookeeper" in the parameter name.

so my suggested parameter name is "serverSelectPolicy", and also rename interface "ChooseServerStrategy" to "ServerSelectStrategy"